validator/db/kv/attestation_history_v2.go

```diff
@@ -127,6 +127,41 @@ func (hd EncHistoryData) SetTargetData(ctx context.Context, target uint64, histo
 	return hd, nil
 }
 
+// UpdateHistoryForAttestation returns a modified attestation history with specified target+epoch pairs marked
```

Comment:
This logic previously existed in validator/client/attest_protect.go and that was the wrong level of abstraction